extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/runtime/MongoClients.java

```diff
@@ -249,6 +250,12 @@ private MongoClientSettings createMongoConfiguration(MongoClientConfig config) {
                 CodecRegistries.fromProviders(providers));
         settings.codecRegistry(registry);
 
+        List<CommandListener> listeners = new ArrayList<>();
```

Comment:
You create a List fo CommandListener in the `getCommandListener` method so this one is not needed.